<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transactionWindow</class>
 <widget class="QMainWindow" name="transactionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transaction</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>18</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditMoney">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxStock"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxAction">
       <item>
        <property name="text">
         <string>Buy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sell</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelMoney">
       <property name="text">
        <string>Money</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAction">
       <property name="text">
        <string>Action</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelStock">
       <property name="text">
        <string>Stock</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonRecord">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewTransaction">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>331</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExecute">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
